stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/actions/ChannelActionsViewModel.kt

```diff
@@ -31,15 +33,7 @@ internal class ChannelActionsViewModel(
                 .await()
                 .data()
                 .let { channelController ->
-                    val membersSource = if (isGroup) {
-                        channelController.members
-                    } else {
-                        Transformations.map(channelController.members) { members ->
-                            members.filter { it.user.id != chatDomain.currentUser.id }
-                        }
-                    }
-
-                    membersSource.observeForever { members ->
+                    channelController.members.observeForever { members ->
```

Comment:
Just noticed we're using `observeForever` here, this seems like it might cause memory leaks

Reply:
Agree, let's refactor that in a separate PR